modules/claims_api/app/controllers/concerns/claims_api/v2/disability_compensation_validation.rb

```diff
@@ -880,9 +873,9 @@ def validate_service_branch_names(service_information)
           unless downcase_branches.include?(sp['serviceBranch'].downcase)
             collect_error_messages(
               source: "/serviceInformation/servicePeriods/#{idx}/serviceBranch",
-              detail: 'serviceBranch must match a service branch ' \
+              detail: "serviceBranch (#{idx}) must match a service branch " \
                       'returned from the /service-branches endpoint of the Benefits ' \
-                      'Reference Data API.'
+                      'Reference Data API.' \
```

Comment:
Obviously not going to hold this up, total nit, but I think the `\` is not needed here?  Did not appear to change the error return for any reason when removing it, again, obviously, total nit.